src/_articles/archive/zones.md

```diff
@@ -302,21 +316,21 @@ streams should have no side effect until listened to.
 A similar situation in synchronous code is the behavior of Iterables,
 which aren't evaluated until you ask for values.
 
-### Example: Using a stream with runZoned()
+### Example: Using a stream with runZonedGuarded()
 
-Here is an example of using a stream with `runZoned()`:
+Here is an example of using a stream with `runZonedGuarded()`:
```

Comment:
**issue:** I understand needing a sentence between the title and the code, but this reads like a rehashing of the title. Is there any way to rephrase or expand upon this so it reads like less of a "I had to do this" sentence?